src/database/DataTypes/NameNote.js

```diff
@@ -15,6 +15,10 @@ export class NameNote extends Realm.Object {
     this._data = JSON.stringify(newValue);
   }
 
+  get patientEventID() {
+    return this.patientEvent?.id;
+  }
```

Comment:
Same as above- so the name note is more similar to those which will be sent from the api